src/reducers/ui.js

```diff
@@ -493,12 +528,24 @@ export const useProgramDimensions = () => {
                 ? [scheduledDateDim]
                 : []),
             incidentDateDim,
-        ].filter(
-            (dimension) =>
-                !!dimension && !hiddenTimeDimensions.includes(dimension.id)
-        )
+        ].filter((dimension) => {
+            // TODO this filter used to return !!dimension && !hiddenTimeDimensions.includes(dimension.id)
```

Comment:
@jenniferarnesen do you want to follow up on this?

Reply:
Resolved I just needed to remove the TODO